src/g1.rs

```diff
@@ -837,6 +847,21 @@ impl AsMut<[u8]> for G1Compressed {
     }
 }
 
+impl ConstantTimeEq for G1Compressed {
+    fn ct_eq(&self, other: &Self) -> Choice {
+        self.0.ct_eq(&other.0)
+    }
+}
+
+impl Eq for G1Compressed {}
+impl PartialEq for G1Compressed {
+    #[inline]
+    fn eq(&self, other: &Self) -> bool {
+        bool::from(self.ct_eq(other))
+    }
+}
```

Comment:
This may be a little out of scope, but supporting PartialEq for the Compressed and Uncompressed types was necessary for properly testing the zeroize support, and I thought it might as well be constant time. Supporting Copy is also necessary for DefaultIsZeroes.

Reply:
If you *want* the type to be `Copy`, use `DefaultIsZeroes`.

If you'd prefer the type not be `Copy`, you can write a `Drop` impl that uses `Zeroize::zeroize` on the inner type instead. This has the advantage that it can leave the type in a state which otherwise wouldn't be considered valid, since it's a `Drop` handler and it doesn't need to respect invariants.

Reply:
I'm fine with this. The reason that the wrappers like `G1Compressed` exist is because we couldn't use `[u8; 48]` directly (due to `LengthAtMost32` restrictions in stdlib). Plain arrays are `Copy`.